addons/binding/org.openhab.binding.innogysmarthome/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,452 @@
+        <category>Temperature</category>
+        <tags>
+            <tag>heating</tag>
+        </tags>
+        <state readOnly="true" />
+    </channel-type>
+
+    <!-- HumiditySensor -->
+    <channel-type id="HumiditySensor_Humidity">
+        <item-type>Number</item-type>
+        <label>Actual Humidity</label>
+        <description>Actual measured room humidity</description>
+        <category>Humidity</category>
+        <tags>
+            <tag>heating</tag>
+        </tags>
+        <state pattern="%.1f%%" readOnly="true">
+        </state>
+    </channel-type>
+        <channel-type id="HumiditySensor_MoldWarning" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Mold Warning</label>
+        <description>Active, if humidity is over a threshold (configured in innogy app)</description>
+        <category>Humidity</category>
+        <tags>
+            <tag>heating</tag>
+        </tags>
+        <state readOnly="true" />
+    </channel-type>
+
+    <!-- WindowDoorSensor Channel -->
+    <channel-type id="WindowDoorSensor">
+        <item-type>Contact</item-type>
+        <label>Contact</label>
+        <description>Shows the open/close state</description>
+        <category>Contact</category>
+        <tags>
+            <tag>switch</tag>
+        </tags>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <!-- SmokeDetectorSensor Channel -->
+    <channel-type id="SmokeDetectorSensor">
+        <item-type>Switch</item-type>
+        <label>Smoke</label>
+        <description>Shows if smoke was detected</description>
+        <category>Smoke</category>
+        <tags>
+            <tag>smoke</tag>
+        </tags>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <!-- AlarmActuator Channel -->
+    <channel-type id="AlarmActuator">
+        <item-type>Switch</item-type>
+        <label>Alarm</label>
+        <description>Switches the alarm on/off</description>
+        <category>Alarm</category>
+        <tags>
+            <tag>smoke</tag>
+        </tags>
+    </channel-type>
+
+    <!-- MotionDetectionSensor Channel -->
+    <channel-type id="MotionDetectionSensor">
+        <item-type>Number</item-type>
+        <label>Motion count</label>
+        <description>The count of detected motions</description>
+        <category>Count</category>
+        <tags>
+            <tag>motion</tag>
+        </tags>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <!-- LuminanceSensor Channel -->
+    <channel-type id="LuminanceSensor">
+        <item-type>Number</item-type>
+        <label>Luminance</label>
+        <description>Shows the detected luminance in percent</description>
+        <category>Light</category>
+        <tags>
+            <tag>luminance</tag>
+        </tags>
+        <state pattern="%d%%" readOnly="true"></state>
+    </channel-type>
+    
+    <!-- PushButtonSensor Channel -->
+    <channel-type id="PushButtonSensor">
```

Comment:
I think not as the innogy pushbutton does not deliver a pressed or released state, but increases a pushed counter.